index.html

```diff
@@ -108,22 +123,23 @@ <h3>I'm a software developer</h3>
         </ul>
       </div>
     </section>
-    <section class="works" id = "portfolio">
+    <section class="works" id="portfolio">
       <div class="recent-works">
-        <div class="title"> 
+        <div class="title">
           <div class="title-h1">
             <h1>My Recent Works</h1>
           </div>
           <div class="line"></div>
         </div>
         <div class="section-2">
-          <div> <img src="img/Placeholder.svg" alt="" /> </div>
-          <div class="multi-post-stories"> 
-             <h2>Multi-Post Stories</h2>
+          <div><img src="img/Placeholder.svg" alt="" /></div>
+          <div class="multi-post-stories">
+            <h2>Multi-Post Stories</h2>
```

Comment:
- Kindly fix the pop-up model functionality that is not working for your first two projects as the previous reviewer asked.
When I press the `see project` button in the first two projects the page is refreshed. The second project is worked but if I pressed the project itself not the `see project` button.